substrafl_examples/get_started/plot_substrafl_torch_fedavg.py

```diff
@@ -210,9 +209,6 @@ def accuracy(datasamples, predictions_path):
 # Specify the machine learning components
 # ***************************************
 #
-# This section uses the PyTorch based SubstraFL API to simplify the definition of machine learning components.
-# However, SubstraFL is compatible with any machine learning framework.
-#
```

Comment:
@ThibaultFy I know you suggested keeping this but it is mentioned in detail at the start of this example so putting it again in the middle seems like overkill

Reply:
I would keep it, even if it's mentioned at the start. I think it's worth saying it again as it is a super important for user to understand it won't bet the same interface with other FL frameworks.

Reply:
Pedagogy is the art of repetition 😄